ethstorage/archiver/guide-devnet.md

```diff
@@ -342,17 +291,81 @@ make op-node
   --rpc.enable-admin \
   --l1=$L1_RPC_URL \
   --l1.rpckind=$L1_RPC_KIND \
-  --l1.beacon=http://localhost:3602 \
-  --l1.beacon-archiver http://localhost:6678
+  --l1.beacon=http://localhost:3602 
 ```
 
 **Note:**
 1. P2P is disabled so that it can only sync data from L1.
 2. The L1 beacon URL is directed to the beacon proxy, where blobs expire quickly.
-3. The beacon archiver is configured to point to the es-node archive service. 
 
-## Verifying Sync Status
+You can observe that the validator node is unable to sync properly with "derivation failed" error due to "failed to fetch blobs". 
+
+Now, navigate to the parent directory in preparation for the next steps.
+
+## Running EthStorage Node
```

Comment:
If I understand correctly, this is the only instance of es-node in the network, and the downloader only attempts to retrieve blobs from the latest finalized block, while the P2P module syncs older blobs from other peers. Since this es-node was launched after the L2 node, there’s a chance that the es-node might fail to sync blobs. I would recommend launching the es-node before starting the L2 node.

Reply:
Yes that makes sense. Changed the order again.